var/spack/repos/builtin/packages/py-entity-management/package.py

```diff
@@ -12,8 +12,7 @@ class PyEntityManagement(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/entity-management'
     git      = 'ssh://bbpcode.epfl.ch/nse/entity-management'
 
-    version('1.0.5', tag='entity-management-v1.0.5', preferred=True)
-    version('0.1.12', tag='entity-management-v0.1.12')
+    version('1.0.9', tag='entity-management-v1.0.9')
```

Comment:
As stated elsewhere, I'm not really a fan of blanket-removal of old versions. For easier reproducibility of issues with older versions, please keep at least a few around.